packages/patternlab/styleguide/source/_patterns/03-organisms/feedback/mass-feedback.twig

```diff
@@ -66,7 +66,7 @@
     </div>
 
     <div class="ma_feedback-fieldset submitButtonWrapper">
-      <input class="submitButton ma__button ma__button--small" type="submit" value="Send Feedback" />
+      <input class="submitButton ma__button ma__button--uppercase ma__button--small" type="submit" value="Send Feedback" />
```

Comment:
```suggestion
      <input class="submitButton ma__button ma__button--uppercase ma__button--xsmall" type="submit" value="Send Feedback" />
```

To match what's in production, we will need to add a classname (see comment earlier)